src/modules/checkbox.js

```diff
@@ -0,0 +1,22 @@
+const current = null;
+class Status {
+  constructor(state = false) {
+    this.state = state;
+  }
+
+  current() {
+    return this.state;
+  }
+
+  on() {
+    this.state = true;
+    return this.state;
+  }
+
+  off() {
+    this.state = false;
+    return this.state;
+  }
+}
+
+export { Status, current };
```

Comment:
I don't see you are using the current val anywhere so please look if is something necessary in your code if no better remove extra code lines.